packages/react-scripts/scripts/eject.js

```diff
@@ -28,8 +29,9 @@ prompt(
 
   console.log('Ejecting...');
 
+  // NOTE: get ownPath and appPath from config/paths.js ?
```

Comment:
You should do this. :)
We can then remove `isReactScriptsLinked.js` and keep the logic in `paths.js`, and you can detect if it's linked by comparing `ownPath` and `appPath` (or similar / other logic).